src/styles/components/_check-list.scss

```diff
@@ -0,0 +1,30 @@
+@font-face {
+  font-family: "gcds-icons";
+  src: url("https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-fonts@0.2.2/icons/gcds-icons.eot");
+  src: url("https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-fonts@0.2.2/icons/gcds-icons.eot#iefix")
+      format("embedded-opentype"), url("https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-fonts@0.2.2/icons/gcds-icons.ttf")
+      format("truetype"),
+    url("https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-fonts@0.2.2/icons/gcds-icons.woff")
+      format("woff"), url("https://cdn.design-system.alpha.canada.ca/@cdssnc/gcds-fonts@0.2.2/icons/gcds-icons.svg")
+      format("svg");
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+.check-list {
+  > li {
+    position: relative;
+    max-width: 65ch;
```

Comment:
Is this supposed to match our maximum character width for text? If yes, could we please use the `--gcds-text-character-limit` token instead of hard-coding the value to avoid potential future misalignment when updating the value?